### DISCUSSION TITLE
Overlapping Found

### DISCUSSION THREAD

**Discussion by ilynivin** [community/community#55830] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have send a Screen shot of  the Bug . So that you can identify the issue 

![bug](https://github.com/community/community/assets/100193482/7388d01b-4203-4232-8b3e-29fad8b91b1f)

**Comment by JaaiDead** [comment]
![image](https://github.com/community/community/assets/98309228/5e85dbdc-552f-4e84-aca4-2539f489de9e)

**Reply by JaaiDead** [reply]
yup

**Reply by ilynivin** [reply]
:(

**Comment by RevertIT** [comment]
Related: https://github.com/orgs/community/discussions/55493

**Comment by kuskebabi** [comment]
in mobile it turns into tablet mode
![Screenshot_2023-05-19-12-14-28](https://github.com/community/community/assets/103432992/9cfb7089-922a-492c-9e74-520278b9b1e9)
